Cinema-Reservation-System-Front-End-Client-Part-Internship-project/src/app/app.config.ts

```diff
@@ -1,8 +1,29 @@
-import { ApplicationConfig, provideZoneChangeDetection } from '@angular/core';
+import {
+  ApplicationConfig,
+  importProvidersFrom,
+  provideZoneChangeDetection,
+} from '@angular/core';
 import { provideRouter } from '@angular/router';
 
 import { routes } from './app.routes';
+import {
+  HTTP_INTERCEPTORS,
+  provideHttpClient,
+  withInterceptorsFromDi,
+} from '@angular/common/http';
+import { HttpErrorInterceptor } from './shared/http-error.interceptor';
```

Comment:
For commonly used folders such as 'shered' add "@shared/*": ["app/shared/*"] to the "paths" property of tsconfig, so u can chage importes from the shared folder to something like 'import ... from "@shered/http-error.interceptor"'